data/ui/properties.ui

```diff
@@ -208,7 +208,7 @@
                     </child>
                     <child>
                       <object class="GtkButton" id="button_properties_open_files">
-                        <property name="label" translatable="yes">Open Files</property>
+                        <property name="label" translatable="yes">Open files</property>
```

Comment:
This could break translations, so I'm not sure if this is a good idea

Reply:
OK, I'll create separate PR then.